apps/etterlatte-saksbehandling-ui/client/src/components/person/oppgavebehandling/oppsummering/FullfoerOppgaveModal.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import { Alert, BodyLong, BodyShort, Button, Heading, Modal } from '@navikt/ds-react'
+import { SaktypeTag } from '~components/nyoppgavebenk/Tags'
+import { ButtonWrapper } from '~shared/modal/modal'
+import { OppgaveDTOny } from '~shared/api/oppgaverny'
+import { isPending, isSuccess, useApiCall } from '~shared/hooks/useApiCall'
+import { opprettBehandling } from '~shared/api/behandling'
+import { NyBehandlingRequest } from '~shared/types/IDetaljertBehandling'
+import { useNavigate } from 'react-router-dom'
+
+interface ModalProps {
+  oppgave: OppgaveDTOny
+  behandlingBehov: NyBehandlingRequest
+}
+
+export default function FullfoerOppgaveModal({ oppgave, behandlingBehov }: ModalProps) {
+  const [open, setOpen] = useState(false)
+  const navigate = useNavigate()
+
+  const [status, opprettNyBehandling] = useApiCall(opprettBehandling)
+
+  const ferdigstill = () => {
+    opprettNyBehandling(
+      {
+        ...behandlingBehov,
+        mottattDato: behandlingBehov!!.mottattDato!!.replace('Z', ''),
+      },
+      () => {
+        setTimeout(() => {
+          navigate(`/person/${oppgave.fnr}`)
+        }, 5000)
```

Comment:
Vilkårlig wait? Antar du her at den er opprettet etter 5 sekunder?

Reply:
Nei, det er for å rekke å vise varsel til saksbehander før de redirectes. Vi har ingen smud måte å gjøre det på i dag, så da laget jeg en liten jallaløsning i mellomtiden for å ikke bruke masse tid på det akkurat nå. 

Når behandling er opprettet OK vises noe á la: "Behandling opprettet, du sendes straks til saksoversikten..."